### PR TITLE
Added a bit more marketing

### PR DESCRIPTION
**Pull request by advanderveer**
We want to make sure users that arrive on the README can easily move to other sources of inspiration

### CODE REVIEW

**Comment by Maaikest** [comment]
Looks good! A few small marketing comments:

In the second paragraph:
_Replace:_ In order to make using these cloud resources more straightforward we’ve designed a CLI that fits your workflow. 
_with:_ In order to make our cloud resources accessible and easy to use, we've developed a CLI that fits your workflow.

Maybe add a title before that previous sentence: Easy to use for engineers, researchers and developers

Then in the first sentence of the paragraph _Docker_, both Nerd and Docker are written in lower case and this should be upper case. 

In the paragraph Documentations, change the bullets to:
[Quickstarts ](https://www.nerdalize.com/docs/quickstarts)- To quickly get you up and running.
[Ready-to-use Software](https://www.nerdalize.com/applications/) - We have application-specific guides for Python or FFmpeg for you to get started.
[CLI Reference ](https://www.nerdalize.com/docs/reference/cli) - For a reference of all available commands

File: README.md [hunk 1]
```diff
@@ -1,73 +1,32 @@
...
-```
---config-file=  location of config file [$NERD_CONFIG_FILE]
---session-file= location of session file [$NERD_SESSION_FILE]
---debug=  show debug output [$NERD_DEBUG]
---output=[pretty|raw|json] Format of the output (default: pretty)
-```
+__Features__:
+  - Moving __datasets__ from you workstation to the cloud and back is included right into the workflow
+  - Nerd ensures efficient and quick datatransfers through a __deduplication__ algorithm
+  - Send in __thousands of jobs__, the Nerd makes your resources are used as efficient as possible
```
Comment (alexanderweiss) [inline README.md]: "the Nerd", or just "Nerd"?

Comment (alexanderweiss) [reply README.md]: makes your -> makes sure your
efficient as possibly -> efficiently as possible

File: README.md [hunk 2]
```diff
@@ -1,73 +1,32 @@
...
+  - Package your software using the industry-standard __Docker__ containers
```
Comment (alexanderweiss) [inline README.md]: using the industry-standard -> using industry-standard (? makes more sense to talk about "containers" than to talk about "the containers", no?)

File: README.md [hunk 3]
```diff
@@ -1,73 +1,32 @@
...
+  - Follows basic CLI conventions to provide a __scriptable__ interface your daily-dose of automation goodness
```
Comment (alexanderweiss) [inline README.md]: interface your daily-dose of -> interface for your daily dose of

File: README.md [hunk 4]
```diff
@@ -1,73 +1,32 @@
...
+
+## Documentation
+To start running your compute on the Nerdalize cloud you'll need to setup an account and download the Nerd CLI itself.
```
Comment (alexanderweiss) [inline README.md]: setup -> set up (or sign up for?)

File: README.md [hunk 5]
```diff
@@ -1,73 +1,32 @@
...
+
+  - [Quickstarts](https://www.nerdalize.com/docs/) - To quickly get you up and running.
+  - [Ready-to-use Software ](https://www.nerdalize.com/applications/) - We have application-specific guides for Python or FFmpeg for you to get started.
+  - [CLI Reference](https://www.nerdalize.com/docs/reference/cli/) - For a reference of all available commands
+
+---
 
 ## Power users
+The Nerd is capable of some powerful scripting and customization options. These guides are somewhat advances and not necessary to comprehend when just starting out on the Nerdalize cloud.
```
Comment (alexanderweiss) [inline README.md]: These guides -> This guide of The rest of this guide did?

**Comment by advanderveer** [comment]
Thanks @alexanderweiss , added your suggestions in master

**Pull request merged**